.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 jspm_packages
 .env
 dist
+config.js
+test-data.js
```

Comment:
a local file i was using to test database queries. not using it anymore so i'll delete it. but is it ok to have these kind of things in .gitignore?